src/pages/TransactionsPage/TransactionsForm/TransactionsForm.js

```diff
@@ -0,0 +1,81 @@
+import { StyledTransactionForm } from "./TransactionsForm.styled";
+import { DatePicker } from "../../../components/form/DatePicker/DatePicker";
+import { TimePicker } from "../../../components/form/TimePicker/TimePicker";
+import { TextInput } from "../../../components/form/TextInput/TextInput";
+import { RadioInput } from "../../../components/form/RadioInput/RadioInput";
+import { DropdownInput } from "../../../components/form/DropdownInput/DropdownInput";
+import { NumberInput } from "../../../components/form/NumberInput/NumberInput";
+import { FormButton } from "../../../components/form/FormButton/FormButton";
+import { useState } from "react";
+
+export const TransactionForm = () => {
+  const [form, setForm] = useState({
```

Comment:
is this value and setter are used to set form as component or set form values? ;d

I would rename it to something like `formValues, setFormValues`